crates/wasmcloud-host/src/dispatch.rs

```diff
@@ -240,7 +240,7 @@ pub enum WasccEntity {
     Capability {
         id: String,
         contract_id: String,
-        binding: String,
+        link: String,
```

Comment:
same, use `link_name`. Apologies for this, but in older versions even `binding` wasn't quite good enough (should've been `binding_name`). This is a result of essentially accumulating "scar tissue" over a year of iterative development.